package-lock.json

```diff
@@ -22017,16 +22018,6 @@
 						"yauzl": "^2.7.0"
 					},
 					"dependencies": {
-						"are-we-there-yet": {
```

Comment:
I spoke to @gziolo about this. I have no idea why it's happening.

Reply:
Try deleting `node_modules` and running `npm install` again.

Reply:
I did that for you in ef9dc987ca0d2e141367255c0eba1c150e850c40 😀